MYR_rails/public/stylesheets/layout/styles/framework.css

```diff
@@ -55,7 +55,7 @@ address{font-style:normal; font-weight:normal;}
 hr{display:block; width:100%; height:1px; border:solid; border-width:1px 0 0 0;}
 
 .wrapper{display:block; width:100%; margin:0; padding:0; text-align:left; word-wrap:break-word;}
-#header, #announce, #breadcrumb, #container, #footer, #copyright{display:block; width:1348px; margin:0 auto;}
```

Comment:
I've updated it to `max-width: 1280px;` which is the widest it can get on a 13'' laptop.
I'm pretty bad in CSS - feel free to close the PR if you think it's not the right direction. I was only looking for a quick way to stop overflowing the browser viewport.

Reply:
Thanks; I think it is an improvement.